### PR TITLE
Upk8s

### LINKED ISSUE #150: Use K8s!

### PR DESCRIPTION
**Pull request by Xiangrui2019**
Danger! Don't merge. There are incompatible changes (about EnforceHttps HSTS)
closed #150

### CODE REVIEW

File: Kahla.Server/appsettings.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     }
   ],
   "ConnectionStrings": {
-    "DatabaseConnection": "Server=(localdb)\\mssqllocaldb;Database=KahlaLocal;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DatabaseConnection": "Data Source=192.168.1.118;Initial Catalog=KahlaLocal;User ID=sa;Password=Xr111900$$$;"
```
Comment (Anduin2017) [inline Kahla.Server/appsettings.json:23]: Changing this may cause my environment stop working.

**Review by Anduin2017 — CHANGES_REQUESTED** [review]

**Comment by Anduin2017** [comment]
So how can we fix the HSTS issue? What is the best practice for K8S to deploy HTTPS only service? Any advice or solution?

**Comment by Xiangrui2019** [comment]
Use Loadbalancer or Ingress Nginx

File: Kahla.Server/appsettings.json (hunk 1 above)
Comment (Xiangrui2019) [reply Kahla.Server/appsettings.json:23]: This change at my working. working end I'm revert it.

**Pull request closed without merging**